### DISCUSSION TITLE
IBMタグの修正について

### DISCUSSION THREAD

**Discussion by takakiyo** [increments/qiita-discussions#335] [要望・アイデア・不具合]
お世話になっております。

IBMというタグについてですが，なぜか1文字目のIが小文字のiで登録されてしまっています。
https://qiita.com/tags/ibm

そのため，このタグがつけられた全ての記事で「iBM」と表示されてしまっています。
商標に関することですので，全て大文字の「IBM」に修正いただけないでしょうか。

**Comment by tsubolabo** [comment]
私もこれは気になっていました。

　これ以外にも、MathWorks 社が自らは MATLAB と名乗っているのに、そのタグ名は matlab なのも引っ掛かっています。しかし、私が参加する以前から、ここには多くの関係者がいるのに、誰も問題提起している様子が見られないことから、それなりの正当な事情があるのだろうと黙っていました。

　今回は、便乗して言わせていただきました。

**Comment by tanaka-asami415** [comment]
@takakiyo @tsubolabo 
ありがとうございます🙏こちら対応させていただきます！
タグの修正リクエストは、次回はこちらhttps://github.com/increments/qiita-discussions/discussions/237 にお願いいたします！🙇

**Reply by wataru86** [reply]
@takakiyo @tsubolabo

IBMとMATLABのタグ名の修正が完了しました！

https://qiita.com/tags/ibm
https://qiita.com/tags/matlab
